widget_driver_test/test/driver_tester_test.dart

```diff
@@ -1,7 +1,45 @@
-import 'package:test/scaffolding.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:widget_driver/widget_driver.dart';
+import 'package:widget_driver_test/src/driver_tester.dart';
+
+class MockWidgetDriver extends WidgetDriver {
+  void triggerNotifyWidgets() {
+    notifyWidget();
+  }
+}
 
 void main() {
-  test('sample test', () {
-    // TODO add tests
+  late MockWidgetDriver mockDriver;
+
+  setUp(() {
+    mockDriver = MockWidgetDriver();
+  });
+
+  testWidgets('should wait for notifyWidget call', (WidgetTester tester) async {
+    final driverTester = DriverTester(mockDriver, tester);
+    Future<void> testFuture = driverTester.waitForNotifyWidget();
+
+    mockDriver.triggerNotifyWidgets();
+
+    await testFuture;
```

Comment:
Its generally nice to have a test verify or expect something.
Could we rewrite the tests to end with an expect.

It could look something like this:

```
testWidgets('should wait for notifyWidget call', (WidgetTester tester) async {
    final driverTester = DriverTester(mockDriver, tester);
    Future<void> testFuture = driverTester.waitForNotifyWidget();

    mockDriver.triggerNotifyWidgets();

    await expectLater(testFuture, completes);
});
```

Reply:
Update on commit 58de41b04c82ba0fa7e37f80538cb45ec8ff4c9f